lms/djangoapps/courseware/tests/test_about.py

```diff
@@ -417,6 +417,32 @@ def test_course_price_is_not_visble_in_sidebar(self):
         self.assertNotIn('<span class="important-dates-item-text">$10</span>', resp.content)
 
 
+@attr(shard=1)
+class AboutSidebarHTMLTestCase(SharedModuleStoreTestCase):
+    """
+    This test case will check the About page for the content in the HTML sidebar.
+    """
+    def setUp(self):
+        super(AboutSidebarHTMLTestCase, self).setUp()
+        self.course = CourseFactory.create()
+
+    def test_html_sidebar_empty(self):
```

Comment:
This doesn't appear to be an "empty" section, as much as "not present".

I read @ziafazal 's comment to ask for DDT-style tests to cover a variety of different possibilities:
- present, but empty
- present, with HTML contents, perhaps with a few different examples
- present with text contents
- etc.

Here, we're only really testing "is the About Sidebar present or not,
which isn't as expansive of a test suite.

@ziafazal Is that what you were getting at or did I misread?

Reply:
I'll look into improving testing tomorrow.